actors/systemsettings/src/main/java/org/sunbird/systemsettings/actors/SystemSettingsActor.java

```diff
@@ -86,8 +85,67 @@ private void setSystemSetting(Request actorMessage) {
     ProjectLogger.log("SystemSettingsActor: setSystemSetting called", LoggerEnum.DEBUG.name());
 
     Map<String, Object> req = actorMessage.getRequest();
+    if (((String) req.get(JsonKey.FIELD)).equalsIgnoreCase(JsonKey.PHONE_UNIQUE)) {
+      checkEmailOrPhoneData(
+          (String) req.get(JsonKey.FIELD),
+          (String) req.get(JsonKey.VALUE),
+          JsonKey.ENC_PHONE,
+          ResponseCode.duplicatePhoneData,
```

Comment:
Combine duplicatePhoneData and duplicateEmailData error codes into one error code (errorDuplicateEntries).